### PR TITLE
chore(translations): UI bus - 2022-05-27

### PR DESCRIPTION
**Pull request by ashrles**
chore(translations): UI bus - 2022-05-27

- Update translations related to: a1347f9 - refactor: format code and remove console log (#538)

### CODE REVIEW

File: src/locale/locales/zh-CN.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "version": "0HFoHIB1i+ZETBO3YI76/g=="
   },
   "SNTable.Pagination.RowsPerPage": {
-    "value": "每页行数",
+    "value": "每页行数：",
```
Comment (LiKang6688) [inline src/locale/locales/zh-CN.json:43]: Hi @ashrles, we would like to remove : here

Comment (mattgengqlik) [reply src/locale/locales/zh-CN.json:43]: will make an bugfix PR for this issue

**Review by mattgengqlik — APPROVED** [review]
LGTM

**Review by HK1112 — COMMENTED** [review]
LGTM

**Review by AntoniaRice — COMMENTED** [review]
LGTM

**Pull request merged**